flink-runtime/src/main/java/org/apache/flink/runtime/asyncprocessing/AsyncExecutionController.java

```diff
@@ -94,19 +103,26 @@ public class AsyncExecutionController<K> {
     final AtomicInteger inFlightRecordNum;
 
     public AsyncExecutionController(MailboxExecutor mailboxExecutor, StateExecutor stateExecutor) {
-        this(mailboxExecutor, stateExecutor, DEFAULT_BATCH_SIZE, DEFAULT_MAX_IN_FLIGHT_RECORD_NUM);
+        this(
+                mailboxExecutor,
+                stateExecutor,
+                DEFAULT_BATCH_SIZE,
+                DEFAULT_BUFFER_TIMEOUT,
+                DEFAULT_MAX_IN_FLIGHT_RECORD_NUM);
```

Comment:
Given that #24657 has been merged, it might be better to verify that the introduced configurations can pass the configured values into AEC through operator setups now.